tests/trace/test_span_json_decoder.py

```diff
@@ -1,59 +0,0 @@
-from phoenix.trace.span_json_decoder import json_to_span
-
-
-def test_span_json_decoder_document_retrieval():
```

Comment:
I'd prefer to preserve the test - just fix the content - this code path does get executed in some of the notebooks.

Reply:
ok